### PR TITLE
Replace the value of __package__ directly in the code

### PR DESCRIPTION
**Pull request by marcelotduarte**
This is an alternative to #609
Runs successfully with pike pdf and don't break samples with pyside2.

### CODE REVIEW

File: cx_Freeze/finder.py [hunk 1]
```diff
@@ -480,6 +483,40 @@ def _LoadPackage(self, name, path, parent, deferredImports, namespace):
...
+            pkg_const_index = len(constants)
+            pkg_name_index = co.co_names.index("__package__")
+            if pkg_const_index > 255 or pkg_name_index > 255:
+                # Don't touch modules with many constants or names;
+                # This is good for now.
+                return
+            # The bytecode/wordcode
+            codes = [LOAD_CONST, pkg_const_index,
+                     STORE_NAME, pkg_name_index]
+            #if pkg_const_index > 255:
```
Comment (anthony-tuininga) [inline cx_Freeze/finder.py]: Does this commented out code need to be present?

Comment (marcelotduarte) [reply cx_Freeze/finder.py]: no.

**Comment by marcelotduarte** [comment]
@anthony-tuininga this is ok?

**Comment by anthony-tuininga** [comment]
I dont see anything obvious so I'm going to assume you've tested this with multiple Python versions and platforms. :-)

**Comment by marcelotduarte** [comment]
Yes, tested in py35/32, 36/32, 37/32, 37/64, 38/64.
Currently, I need it only for pikepdf, but is a generic approach.

**Pull request merged**